src/App.jsx

```diff
@@ -8,16 +8,11 @@ import { ThemeProvider, createTheme } from "@mui/material/styles";
 import { CssBaseline } from "@mui/material";
 import Login from "./components/Login";
 import { useSelector } from "react-redux";
-import store from "./state/store/configureStore";
-import i18n from "./i18n";
+import ParticipantList from "./components/ParticipantList";
 
 const App = () => {
   const { authenticated } = useSelector((state) => state);
-
-  if (navigator.language.includes("sv")) {
```

Comment:
Does the navigation test still go green without this code?

Reply:
It does, this is left over from trying to read the browsers default language and should have been removed before